pkg/image/api/validation/validation_test.go

```diff
@@ -315,24 +315,6 @@ func TestValidateImageStreamMappingNotOK(t *testing.T) {
 			field.ErrorTypeRequired,
 			"image.metadata.name",
 		},
-		"invalid repository pull spec": {
```

Comment:
@miminar This test is called the `TestValidateImageStreamMappingNotOK`. So, this case can't be here anyway.